src/clj_discord_bot/core.clj

```diff
@@ -1,84 +1,64 @@
-      (discord/answer-command data "!d20" (str "You rolled: " (+ (rand-int 100) 1))))
-
-(defn img-search [query]
-      (let [result (http-client/get (img-search-url) {:query-params {"q" query
-                                                                     "num" 1
-                                                                     "searchType" "image"
-                                                                     "start" (+ (rand-int 100) 1)
-                                                                     "key" google-token},
-                                                      :as :json, :debug true})]
-           (get (nth (get-in result [:body :items]) 0) :link)))
-
-(defn find-img
-      "!img <your_query> - Finds a random image of <your_query>"
-      [type data]
-
-      (let [message (get data "content")]
-           (if (.startsWith message "!img ")
-             (discord/answer-command data message (img-search (subs message (count "!img ")))))))
-(defn quaggan-answer [data message]
-      (Thread/sleep (* (+ (rand-int 5) 1) 60 1000))
-      (discord/answer-command data message (img-search "quaggan")))
-
-(defn quaggan-joe [type data]
-      (let [message (get data "content")
-            mentions (get data "mentions")
-            mention-all (get data "mention_everyone")]
-           (println "THIS IS A TEST")
-           (println message)
-           (println quaggan-token)
-
-           (if (.contains message quaggan-token)
-             "test")))
-
-(defn gandhi-spellcheck [type data]
-      (let [message (get data "content")]
-           (if (re-find #"(?i)ghandi" message)
-             (discord/answer-command data message "Gandhi  (╯°□°）╯︵ ┻━┻"))))
-
-(defn links-mentioned [type data]
-      (let [message (get data "content")]
-           (if (re-find #"(?i)link" message)
-             (discord/answer-command data message (img-search "lynx")))))
 
 (defn help [type data]
-      (let [message (get data "content")]
-           (print "helpin")
-           (if (.equals "!help" message)
-             (discord/answer-command data message
-                                     (str "Commands" (apply str
-                                          (map #(str "\n" (:doc (meta %))) [#'find-img
-                                                                            #'d20
-                                                                            #'help])))))))
-
+  (discord/answer-command data
+                          (get data "content")
+                          (str "Commands"
+                               (apply str (map #(str "\n" (:doc (meta %)))
+                                               [#'img-search/find-img
+                                                #'roll/d20
+                                                #'summon/game-summon])))))
+
+(defn game_update [type data]
+  (let [server_id 0 ;server id is not returned in message data, so ignore for now ... (get-in data ["guild_id"])
```

Comment:
are variables normally done with `-` instead of `_`? I gots no idea

Reply:
me neither, i wonder if there is a decent formatter for clojure as well, there is in VS Code but nothing in intellj i think :/